homework.md

```diff
@@ -4,77 +4,78 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is useful for adding style to and changing the look of html elements.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language that enables you to add complexity to web pages.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. It enables the user to monitor and order the different versions of a project more easily.
+2. It makes it easier to collaborate on a project.
+3. Github provides a complete environment/platform that allows anyone to work together from anywhere.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make an exact copy for yourself.
```

Comment:
Hello John, I believe the answer should be "You `fork` a repository so you can make changes to it without affecting the original repository". 
Because `Cloning` means making an exact copy.